core/src/components/infinite-scroll/readme.md

```diff
@@ -12,6 +12,21 @@ The `ion-infinite-scroll` component has the infinite scroll logic. It requires a
 
 Separating the `ion-infinite-scroll` and `ion-infinite-scroll-content` components allows developers to create their own content components, if desired. This content can contain anything, from an SVG element to elements with unique CSS animations.
 
+## Usage with Virtual Scroll
+
+Infinite scroll requires a scroll container to function. When using a virtual scrolling solution, you will need to disable scrolling on the `ion-content` and indicate which element container is responsible for the scroll container with the `.ion-content-scroll-host` class target.
+
+```html
+<ion-content scroll-y="false">
+  <virtual-scroll-element class="ion-content-scroll-host">
+    <!-- Your virtual scroll content -->
+  </virtual-scroll-element>
+  <ion-infinite-scroll (ionInfinite)="loadData($event)">
```

Comment:
Sorry didn't catch this before approval. But we are mixing JavaScript + Angular syntax here. Maybe this is better suited for the `usage` directory files?

It should be `[scrollY]="false"` in Angular, but it seems like we are trying to be framework-agnostic. In that case, `(ionInfinite)` would not be appropriate here.

Reply:
Good catch!